services/repository-managers/src/main/java/org/sagebionetworks/repo/manager/principal/PrincipalManagerImpl.java

```diff
@@ -94,6 +105,10 @@ public void newAccountEmailValidation(NewUser user, String portalEndpoint, Date
 		if (!principalAliasDAO.isAliasAvailable(user.getEmail())) {
 			throw new NameConflictException("The email address provided is already used.");
 		}
+		if (emailQuarantineDao.getQuarantinedEmail(user.getEmail()).isPresent()) {
+			LOG.warn("Cannot send account validation to quarantined email address {}", user.getEmail());
+			return;
```

Comment:
throw an exception informing the use there is a problem with the provided email and to contact support.